hrpsys_ros_bridge/scripts/default_robot.launch.in

```diff
@@ -8,6 +8,7 @@
   <arg name="PROJECT_FILE" default="$(find @PROJECT_PKG_NAME@)/models/@ROBOT@.xml" unless="$(arg NOSIM)"/>
   <arg name="SIMULATOR_NAME" default="@ROBOT@(Robot)0" />
   <arg name="corbaport" default="15005" />
+  <arg name="managerport" default="15006" />
```

Comment:
please add
```
  <!-- available in Kinetic -->
  <!-- <arg name="managerport" default="$(eval arg('corbaport') + 1)"/> -->
```
too, and I think 2810 is metter for default values for manager  until https://github.com/fkanehiro/hrpsys-base/pull/1253/files is merged and released